app/src/main/java/com/github/se/wanderpals/model/viewmodel/AdminViewModel.kt

```diff
@@ -22,7 +22,8 @@ open class AdminViewModel(
   open var listOfUsers = MutableStateFlow(emptyList<User>())
   // delete a user from the database
   open fun deleteUser(userId: String) {
-    viewModelScope.launch { tripsRepository.removeUserFromTrip(userId, tripId) }
+    viewModelScope.launch { tripsRepository.removeUserFromTrip(tripId, userId) }
```

Comment:
Maybe if the current user inside of the session manager is the userId that is being removed (you are leaving the trip), you should navigate to the overview in addition to doing this